src/perl/Permabit/UserModule.pm

```diff
@@ -37,6 +37,27 @@ sub load {
   return $loaded;
 }
 
+###############################################################################
+# @inherit
+##
+sub loadFromFiles {
+  my ($self) = assertNumArgs(1, @_);
+  my $machine = $self->{machine};
+  my $modFileName = $self->{modFileName};
+
+  # if useUpstream, download vdo package from distribution repo
+  if ($self->{useUpstream}) {
+    my $topdir = makeFullPath($machine->{workDir}, $self->{modVersion});
+    $self->_step(command => "mkdir -p $topdir");
+    my $getFromDnf = join(' ',
+                          "dnf", "download", "--destdir", "$self->{modDir}",
+                          "$modFileName"
+	                 );
```

Comment:
The alignment is weird in theis function call. Wrap the lines at a reasonable length, and don't put ); on its own line.